pages/index.jsx

```diff
@@ -8,10 +8,14 @@ import Pontoon from '../components/pontoon.jsx';
 var Index = React.createClass({
   render: function() {
     var metaData = this.props.metaData;
+    var robots = 'index, follow';
     var googleFonts = "https://fonts.googleapis.com/css?family=Open+Sans:600,400,300,300italic";
     if (this.props.localeInfo === "cs") {
       googleFonts += "&subset=latin-ext";
     }
+    if (metaData.current_url.indexOf('thank-you') !== -1) {
```

Comment:
You could also flip you variable values, and do a === -1 check. So instead of asking is it not not in the string, you are asking if it's not in the string, which is not not a single negative :P